### PR TITLE
setup and mobile first

### PR DESCRIPTION
**Pull request by StevenWafeek**
In this project we :
- Add HTML file
- Add header to my portfolio
- add a headline section to my portfolio
- Add CSS file with styles
- Add readme and .gitignore files
- Add linter to my project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    
+             <div class="burger">
+                    <img src="picture/Enabled.png" alt="">
+                  </div>
+        </nav>
+
+    </header>
+        <section>
+
+            <nav class="headline">
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Please make sure to have only one `nav` on the page which is contained inside the `header`. You can use other semantic containers or `div` for anything else.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,500;0,600;0,800;1,100&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav>
+            <!-- <img src="picture/header-shapes mobile.png" alt=""> -->
+
+            <div class="logo">
+                <p>Logo</p>
+            </div>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Please edit the logo to be an `a` tag which can be used later as a link to the home page.
```
     <p>
        <a href="#">My Logo</a>
     </p>
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,500;0,600;0,800;1,100&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav>
+            <!-- <img src="picture/header-shapes mobile.png" alt=""> -->
```
Comment (ShadyShawkat) [inline index.html]: - [ ]  Great job so far. However, there's little issue with the layout here. It is only the hamburger that should be inside the nav tag. For example:
```
<header>
   <div class="logo">
      <a href="#">Logo</a>
   </div>
   <nav>
      <img ... />
   </nav>
</header>
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                    <a href="https://www.linkedin.com/in/steven-wafeek-b629341b6/">
+                        <i class="fa-brands fa-linkedin-in"></i>
+                        </a>
+                        <a href="https://twitter.com/steven_hakeem">
+                             <i class="fa-brands fa-github"></i>
+                            </a>
+                            <a href="#">
+                        <i class="fa-regular fa-hand-peace fa-flip-horizontal"></i>
+                    </a>
+                </div>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Kindly implement the list of social icons to be in a list instead, you can use something like the following:
```
<ul>
   <li>
      <a>
         <i>...</i>
      </a>
   </li>
   <li>
      <a>
         <i>...</i>
      </a>
   </li>
   <li>
      <a>
         <i>...</i>
      </a>
   </li>
   ...
</ul>
```

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @StevenWafeek, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @StevenWafeek, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**